docs/workflow/requirements/windows-requirements.md

```diff
@@ -46,7 +46,11 @@ A `.vsconfig` file is included in the root of the _dotnet/runtime_ repository th
 
 ### Build Tools
 
-These steps are required only in case the tools have not been installed as Visual Studio **Individual Components** (described above).
+These steps are required only in case the tools have not been installed as Visual Studio **Individual Components** (described above). You can install the minimum prerequisites using the following winget command:
+
+```shell
+winget install --accept-package-agreements cmake ninja-build.ninja python git.git
```

Comment:
People who install cmake and ninja separately from Visual Studio tend to run into weird build errors caused by version mismatch issues eventually. It is best to use cmake and ninja that comes with Visual Studio. It is guaranteed to work with the rest of Visual Studio.

It is certainly fine to add this, but it leads people to the pit of failure.

Reply:
I wasn't even aware that VS was bundling cmake and ninja and would always install via winget. I think we can close this.